### PR TITLE
Swpg6 speak 269 internationalization

### PR DESCRIPTION
**Pull request by sgatnar**
implemented i18n

### CODE REVIEW

File: frontend/lib/pages/base/home/user_menu.dart [hunk 1]
```diff
@@ -39,7 +39,8 @@ class _UserMenuState extends State<UserMenu> {
     );
   }
 
-  Widget buildHeader(BuildContext context,AppLocalizations appLocale) => Container(
+  Widget buildHeader(BuildContext context, AppLocalizations appLocale) =>
```
Comment (nomoruyi) [inline frontend/lib/pages/base/home/user_menu.dart:42]: "_buildBlaBla{ return ... }" anstatt "_buildBlaBla => ..." bei build Methoden verwenden

File: frontend/lib/pages/start/login/login_page.dart [hunk 2]
```diff
@@ -12,82 +11,142 @@ class LoginPage extends StatefulWidget {
...
+          loginText: appLocale.login,
+          press: () {
+            Navigator.popAndPushNamed(context, "register");
+          },
         ),
-        child: Column(
+        const Padding(
+          padding: EdgeInsets.only(top: 5, bottom: 5),
+          child: Divider(
+            color: Colors.black54,
```
Comment (nomoruyi) [inline frontend/lib/pages/start/login/login_page.dart:130]: Speaq Farbpalette nutzen

**Comment by nomoruyi** [comment]
@sgatnar Please fix merge conflicts

File: frontend/lib/pages/base/home/user_menu.dart (hunk 1 above)
Comment (sgatnar) [reply frontend/lib/pages/base/home/user_menu.dart:42]: done

File: frontend/lib/pages/start/login/login_page.dart (hunk 2 above)
Comment (sgatnar) [reply frontend/lib/pages/start/login/login_page.dart:130]: done

**Review by nomoruyi — APPROVED** [review]

**Pull request merged**